contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -10,13 +10,15 @@ import "deps/@openzeppelin/contracts-upgradeable/utils/ReentrancyGuardUpgradeabl
 
 import "interfaces/badger/ISwapStrategyRouter.sol";
 import "interfaces/bridge/IGateway.sol";
+import "interfaces/badger/ISett.sol";
 
 contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     using SafeMathUpgradeable for uint256;
     using SafeERC20 for IERC20;
 
     IERC20 public renBTC;
     IERC20 public wBTC;
+    mapping(address => bool) public approvedVaults;
```

Comment:
You can't do this. This will overwrite the variables below. The contract storage layout is only extendable at the bottom of all storage variables. 

https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies#storage-collisions-between-implementation-versions

Reply:
I was under the implementation this was the first version where it would not brake the storage but I'll be sure to fix that